### DISCUSSION TITLE
v up no response

### DISCUSSION THREAD

**Discussion by vajrabisj** [vlang/v#15171] [General]
on windows, when call v up, there is no response after showing Updating V...

**Comment by Wajinn** [comment]
You will need to provide more details, so that people could possibly help.  Also, possibly refer to installation issues as well (Windows thread), to make sure it's not something there (https://github.com/vlang/v/discussions/categories/installation-issues).

**Reply by vajrabisj** [reply]
Noted, but it's just no responding, after enter 'v up', long-time no responding, no any msg except for 'updating v...' i have to ctr-c to quit.
another thing is i can use all other v commands, except for c2v.... the detail is following from which you can see i have clang on windows, but c2v cannot find the path. cannot use c2v such a good tool really frustrated...

mb@W1 ~
$ clang
clang-8: error: no input files

mb@W1 ~
$ c2v
V panic: cannot find clang in PATH
v hash: 12d57e8
C:/Users/mb/AppData/Local/Temp/v_0/c2v.9898216623934814009.tmp.c:10930:
at _v_panic: Backtrace
C:/Users/mb/AppData/Local/Temp/v_0/c2v.9898216623934814009.tmp.c:28503:
by main__find_clang_in_path
C:/Users/mb/AppData/Local/Temp/v_0/c2v.9898216623934814009.tmp.c:32772:
by _vinit
C:/Users/mb/AppData/Local/Temp/v_0/c2v.9898216623934814009.tmp.c:32784:
by wmain
00569c38 : by ???
00569d9b : by ???
7ff9f5667c24 : by ???

mb@W1 ~
$

**Reply by vajrabisj** [reply]
finally, after long long waiting i got response for 'v up':

Updating V...
From https://github.com/vlang/v
 * branch                master     -> FETCH_HEAD
fatal: refusing to merge unrelated histories
